Kernel/ProcessSpecificExposed.cpp

```diff
@@ -280,7 +280,7 @@ ErrorOr<void> Process::procfs_get_binary_link(KBufferBuilder& builder) const
     auto const* custody = executable();
     if (!custody)
         return Error::from_errno(ENOEXEC);
-    return builder.append(custody->absolute_path().bytes());
+    return builder.append(TRY(custody->try_serialize_absolute_path())->view());
```

Comment:
Not saying we should do it in this PR, but it seems like we should have a way for the `Custody` to serialize directly to the `KBufferBuilder`, instead of having to allocate a `KString` and throw it away?

Reply:
Sure, that sounds like a reasonable idea, this PR is more of a "kill all String in the Kernel crusade", optimizing it would probably be nicer in a future PR :)